html/user/bolt_sched.php

```diff
@@ -326,7 +326,7 @@ function show_next($iter, $view) {
             //
             show_refresh_finished();
             $refresh->update('count=count+1');
-            break;
+            return;
```

Comment:
I have no full picture of what PHP's quirky use of `break` outside a loop was doing in different contexts, nor do I know what it was meant to do in this context. AFAIK, `break` outside a loop exits a script: what does that say about using it in a function like this? Yes, it could mean `return` but I don't know.

Bottom line: this should be reviewed by David who knows the intended semantic of this code.

Otherwise I'm fine with the changes.

Reply:
@davidpanderson Can you recall why you put a `break` here? I don't have the means to test this part.

Reply:
That `break` would have behaved the same as `return` in this case (no PHPun intended). Both calls to `show_next()` are made inside of `case` statements and are immediately followed by a `break`, so if execution was reaching that drifter of a `break` inside the function, and this was not generating a fatal runtime error about `break` being called out of scope, it would just cause execution to break out of the parent `switch` in the calling routine (which it was going to do immediately, anyway, upon return). No need to hold up this PR, in my opinion.